app/components/Search/ResourcesTable.js

```diff
@@ -138,19 +124,14 @@ class ResourcesTable extends Component {
   }
 
   componentDidMount() {
-    if(this.props.userLocation) {
-      let position = this.props.userLocation;
-      let userLocation = {lat: position.coords.latitude, lng: position.coords.longitude};
-      this.setState({
-        location: userLocation
-      });
-      this.loadResourcesFromServer(userLocation);
-    } else {
-      let self = this;
-      this.getLocationGoogle()
-        .then(loc => {
-          self.loadResourcesFromServer(loc);
-        });
+    if (this.props.userLocation) {
```

Comment:
I'm not sure if it's safe to take this out. `loadResourcesFromServer` doesn't do a null check before trying to access the `lat` and `lng` of the location, but even if it did, if we decide to expand to more cities, it'll be confusing for the API to return results from everywhere.